src/components/document/sort-work-document-area.tsx

```diff
@@ -10,15 +10,21 @@ import { ENavTab } from "../../models/view/nav-tabs";
 import { isExemplarType } from "../../models/document/document-types";
 import { ExemplarVisibilityCheckbox } from "./exemplar-visibility-checkbox";
 import { DocumentLoadingSpinner } from "./document-loading-spinner";
+import { DocumentScroller } from "./document-scroller";
+import { DocumentGroup } from "../../models/stores/document-group";
 
 import EditIcon from "../../clue/assets/icons/edit-right-icon.svg";
 import CloseIcon from "../../../src/assets/icons/close/close.svg";
+import ToggleDocumentScrollerIcon from "../../../src/assets/show-hide-thumbnail-view-small-icon.svg";
+import SwitchDocumentIcon from "../../assets/scroll-arrow-small-icon.svg";
 
 interface IProps {
+  openDocumentsGroup?: DocumentGroup;
   openDocumentKey: string;
```

Comment:
This `openDocumentKey` property is redundant with the use of `persistentUI.openSubTabDocument(ENavTab.kSortWork, ENavTab.kSortWork, newKey)`. This will make maintaining the code difficult with two places tracking the opened document key. 

If possible the `openDocumentKey` property should be removed, and the component should just use and observe the state in the `persistentUI`.